--- wm.c
+++ wm.c
@@ -34,6 +34,7 @@
 	Window wnd;
 	XftDraw* xftdraw;
 	GC gc;
+	int batt;
 } silly_bar;

 typedef struct silly_window {
@@ -114,6 +115,7 @@
 	XSelectInput(dpy, bar->wnd, ExposureMask);
 	XMapWindow(dpy, bar->wnd);
 	bar->gc = XCreateGC(dpy, bar->wnd, 0, NULL);
+	bar->batt = open(BATTERY_LOC, O_RDONLY);
 
 	return bar;
 }
@@ -153,11 +155,18 @@
 	if (!title) XFetchName(dpy, focus, &title);
 	XSetErrorHandler(h);
 
+	char bat_str[4]; // 0 - 100
+	lseek(bar->batt, 0, SEEK_SET);
+	read(bar->batt, &bat_str, 3);
+	for (int i = 0; i < 4; i++)
+		if (bat_str[i] < '0' || bat_str[i] > '9') bat_str[i] = 0;
+
 	char status[128];
 	sprintf(status,
-		"%02d:%02d %02d/%02d/%02d",
+		"%02d:%02d %02d/%02d/%02d | %s%%",
 		tm->tm_hour, tm->tm_min,
-		tm->tm_mday, tm->tm_mon + 1, tm->tm_year + 1900
+		tm->tm_mday, tm->tm_mon + 1, tm->tm_year + 1900,
+		bat_str
 	);
 	silly_draw_bar(bar, title ? title : "(desk)", status);
 	if (title) XFree(title);
@@ -166,6 +175,7 @@
 void silly_destroy_bar(silly_bar* bar) {
 	XFreeGC(dpy, bar->gc);
 	XDestroyWindow(dpy, bar->wnd);
+	close(bar->batt);
 	free(bar);
 }
 
